borgmatic/hooks/postgresql.py

```diff
@@ -226,6 +226,11 @@ def restore_database_dump(database_config, log_prefix, location_config, dry_run,
         + (tuple(database['restore_options'].split(' ')) if 'restore_options' in database else ())
         + (() if extract_process else (dump_filename,))
     )
+
+    if database['schemas']:
+        for schema in database['schemas']:
+            restore_command += ('--schema', schema)
```

Comment:
If you wanted to, this could be worked into the `restore_command` declaration above instead of extending the command here. Something like:

```python
  + tuple(itertools.chain.from_iterable(('--schema', schema) for schema in database['schemas']))
```

(Do not feel strongly.)

Reply:
Nice!